trip_journal_app/static/javascript/login_by_mail.js

```diff
@@ -0,0 +1,127 @@
+window.onload = function(){
```

Comment:
By doing so, you override whatever onload handler there might already be. Use addEventListener.